sdk/ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_16.scala

```diff
@@ -8,11 +8,8 @@ import com.daml.ledger.api.tls.TlsConfiguration
 
 package object v1_16 {
   def default(timeoutScaleFactor: Double): Vector[LedgerTestSuite] =
-    v1_15.default(timeoutScaleFactor)
+    v1_15.default(timeoutScaleFactor) ++ Vector(new UpgradingIT)
```

Comment:
Have you verified that this doesn't fail? The comment before implied it did

Reply:
Suggested by @tudor-da, so it should be fine.